packages/core/strapi/lib/services/entity-service/index.d.ts

```diff
@@ -54,14 +55,19 @@ export interface EntityService {
     pagination: PaginationInfo;
   }>;
 
-  findWithRelationCounts<K extends keyof AllTypes, T extends AllTypes[K]>(
+  findWithRelationCountsPage<K extends keyof AllTypes, T extends AllTypes[K]>(
     uid: K,
     params: Params<T>
   ): Promise<{
     results: T[];
```

Comment:
Should this become `data: T[];`

Reply:
I don't think so, it's not the route response but the function response so there isn't the format `data`.
Are you referring to the function `findWithRelationCounts` line 66?

What I did was:
- Renaming `findWithRelationCounts` to `findWithRelationCountsPage` as it was using the pagination mode (like `findPage` is the equivalent of `findMany` but with pagination) but didn't have this information in the function name.
- Creating the function `findWithRelationCounts` that doesn't use the pagination mode